pkg/apis/core/types.go

```diff
@@ -4900,7 +4900,11 @@ type NodeRuntimeHandlerFeatures struct {
 	// +featureGate=RecursiveReadOnlyMounts
 	// +optional
 	RecursiveReadOnlyMounts *bool
-	// Reserved: UserNamespaces *bool
+	// UserNamespaces is set to true if the runtime handler supports UserNamespaces.
+	// More specifically, the underlying runtime is expected to support idmapped mounting.
+	// +featureGate=UserNamespacesSupport
+	// +optional
+	UserNamespaces *bool
```

Comment:
Getting API approval is hard. Do you think it would be better to carve-out those changes into a separate PR?

Reply:
https://github.com/kubernetes/kubernetes/pull/126034